Eatery/Controllers/BRB/BRBLoginViewController.swift

```diff
@@ -121,12 +80,13 @@ class BRBLoginViewController: UIViewController {
         setUpErrorView()
         setUpNetidViews()
         setUpPasswordViews()
-        setUpSaveLoginInfoViews()
         setUpLoginViews()
+    }
 
-        if let (netid, password) = BRBAccountSettings.loadFromKeychain(), !netid.isEmpty, !password.isEmpty {
-            netidTextField.text = netid
-            passwordTextField.text = password
+    override func viewWillAppear(_ animated: Bool) {
+        if !(netidTextField.text?.isEmpty ?? true) && !(passwordTextField.text?.isEmpty ?? true) {
+            netidTextField.text = nil
+            passwordTextField.text = nil
```

Comment:
This if statement is kind of complicated. Could you add a short explanation of what the condition is? Even better: could you make it simpler?